sources/packages/api/src/route-controllers/education-program/education-program.controller.ts

```diff
@@ -48,6 +51,26 @@ export class EducationProgramController {
     }));
   }
 
+  @Get(":programId")
```

Comment:
The same method was already merged. Please take a look at the one right now on main.